docs/docs/deploy/prerequisites.md

```diff
@@ -34,8 +34,9 @@ to do so:
 
 ## Security Group and Firewall Settings
 
-By default, Firezone requires ports `80/tcp`, `443/tcp`, and `51820/udp` to be open.
-This can change based on what you've configured in the configuration file.
+By default, Firezone requires ports `80/tcp` or `443/tcp`, and `51820/udp` to be
+accessible. We highliy recommend using https over `443/tcp`.
```

Comment:
I think we can drop the `80/tcp` requirement as it's not *required* by default. For clarification, this could be

> By default, Firezone requires ports `443/tcp` and `51820/udp` traffic to be accessible for HTTPS and WireGuard traffic respectively.